app/login/page.tsx

```diff
@@ -130,7 +153,7 @@ const LoginPage: React.FC = () => {
                     onChange={handleInputChange}
                     required
                 />
-                <button type="submit">Login</button>
+                <button type="submit" disabled={(loading || attempts > 5) ? true : false}>{attempts}{loading ? 'Logging in ...' : 'Login'}</button>
```

Comment:
Looks like you're always showing the number of attempts here. Is that for debug?

Reply:
Oh thanks for catching that Brian :) Yes, that was for debugging.